LBH.AdultSocialCare.Api/V1/Infrastructure/DatabaseContext.cs

```diff
@@ -51,12 +53,42 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             // Seed term time consideration options
             modelBuilder.ApplyConfiguration(new TermTimeConsiderationOptionsSeed());
 
+            // Seed day care how long options
+            modelBuilder.ApplyConfiguration(new OpportunityLengthOptionsSeed());
+
+            // Seed day care how many times per month options
+            modelBuilder.ApplyConfiguration(new OpportunityTimesPerMonthOptionsSeed());
+
             // Seed home care service types
             modelBuilder.ApplyConfiguration(new HomeCareServiceTypesSeed());
-            modelBuilder.ApplyConfiguration(new HomeCareServiceTypeMinutesSeed());
+            // modelBuilder.ApplyConfiguration(new HomeCareServiceTypeMinutesSeed());
```

Comment:
This seed will be needed for the home care config

Reply:
> This seed will be needed for the home care config

I did this testing if seed was reason for migration error. Forgot to uncomment. Will do